Lesson5/src/app/pages/User.js

```diff
@@ -1,27 +1,23 @@
-import React from 'react';
+import React, { useState, useEffect } from "react";
 import UserProfile from '../components/User';
 import axios from 'axios';
 
-export default class User extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      user: null
-    };
-  }
+const User = props => {
+  const [userS, setUserS] = useState({});
```

Comment:
```suggestion
  const [userS, setUserS] = useState(null);
```
чтобы код указанный ниже ( `{userS && <UserProfile {...userS} />}` ) работал корректно:)